### DISCUSSION TITLE
MCT001 new (blank) app (EDIT: not an issue)

### DISCUSSION THREAD

**Discussion by get-flat** [CommunityToolkit/Maui#839] [Q&A] [answered]
Closed

**Comment by get-flat** [comment, marked as answer]
@brminnick thank your for converting the issue.
Could you maybe explain why exactly it is not a bug?

**Reply by TheCodeTraveler** [reply]
Of course! I converted this to a Q&A Discussion because this Issue only asked a question:
> Is there any workaround to dismiss MCT001?

It didn’t describe a bug. It didn’t describe a problem.

**Comment by CliffAgius** [comment]
Closed as answered.
